openfl-docker/licenses.sh

```diff
@@ -33,8 +33,8 @@ if [ "$INSTALL_SOURCES" = "yes" ]; then
     rm -rf /var/lib/apt/lists/*
 fi
 
-mkdir -p thirdparty
-cd thirdparty
+mkdir -p third-party
+cd third-party
```

Comment:
I guess could be reverted/skipped here but I thought "why not" if it is just a bundle of licenses?

Reply:
Yes, let's revert this change as it may have an impact on downstream projects that use OpenFL